.trivyignore

```diff
@@ -0,0 +1,10 @@
+# Docs: https://aquasecurity.github.io/trivy/latest/docs/configuration/filtering/#trivyignore
+
+# In docker files:
+# non-root user is always created within docker, but we switch it only in CI workflows;
+# not enforcing non-root user makes it easier for developers to use their own users in local container
+AVD-DS-0002
+
+# In docker files:
+# HEALTHCHECK is not required for development, nor in CI (failed docker = failed CI)
```

Comment:
PSE approval is required for:
Hadolint YAML configuration files that disable any built-in rules. (I assume the same requirement will apply to Trivy)

Reply:
Since this task is blocking the other, let's just merge it as it is. There is still a lot of time left until the release, so we will certainly be able to address these problems later.

Reply:
I created an issue to follow this up: https://github.com/oneapi-src/unified-memory-framework/issues/262

Reply:
@wlemkows I am closing this conversation as we have agreed on the next steps